pymatgen/entries/computed_entries.py

```diff
@@ -11,26 +11,187 @@
 """
 
 import json
+import abc
 
-from monty.json import MontyEncoder, MontyDecoder
+from monty.json import MontyEncoder, MontyDecoder, MSONable
 
 from pymatgen.core.composition import Composition
 from pymatgen.core.structure import Structure
 from pymatgen.entries import Entry
 
-__author__ = "Shyue Ping Ong, Anubhav Jain"
+__author__ = "Ryan Kingsbury, Shyue Ping Ong, Anubhav Jain"
 __copyright__ = "Copyright 2011, The Materials Project"
 __version__ = "1.1"
 __maintainer__ = "Shyue Ping Ong"
 __email__ = "shyuep@gmail.com"
 __status__ = "Production"
-__date__ = "Apr 30, 2012"
+__date__ = "April 2020"
+
+
+class EnergyAdjustment(MSONable):
+    """
+    Lightweight class to contain information about an energy adjustment or
+    energy correction.
+    """
+    def __init__(self, value, name="Manual adjustment", cls="None", description=""):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+            name: str, human-readable name of the energy adjustment.
+                (Default: Manual adjustment)
+            cls: str, the name of the Compatibility class used to generate the
+                energy adjustment. (Default: None)
+            description: str, human-readable explanation of the energy adjustment.
+        """
+        self._value = value
```

Comment:
No need to take in a value here. `EnergyAdjustmet` just cares about name, cls, description and that any implementation provides a value prop of some type.